keyboards/handwired/aim65/keymaps/bonnee/keymap.c

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2021 Matteo Bonora
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define CTESC LCTL_T(KC_ESC)
+
+enum layer {
+    BL = 0,
+    FL = 1
```

Comment:
The numbering isn't needed, actually. It starts at 0, and automatically increments.
```suggestion
    BL,
    FL
```

Reply:
Right! numbering was from when I used defines